### PR TITLE
Feature/vmo 6859/vmo 7197/autolink btn and panel

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 1]
```diff
@@ -318,7 +318,7 @@ import {BModal, BootstrapVue, BTooltip} from 'bootstrap-vue'
 import Vue from 'vue'
 import Lang from '@/lib/filters/lang'
 import Permissions from '@/lib/mixins/Permissions'
-import Routes, {routeFrom} from '@/lib/mixins/Routes'
+import Routes from '@/lib/mixins/Routes'
```
Comment (bulatgab) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:321]: unrelated, cleanup

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 2]
```diff
@@ -628,7 +628,6 @@ export class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang) {
   @Getter isResourceViewerEnabled?: boolean
   @Mutation setTreeSaving!: (isSaving: boolean) => void
   @Action attemptSaveTree!: void
-  @Getter getToolbarConfig!: boolean
```
Comment (bulatgab) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:631]: unrelated, cleanup

File: vue.config.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ const cookieParser = require('cookie-parser')
 module.exports = {
   lintOnSave: false,
   runtimeCompiler: true,
+  parallel: false,
```
[the remark this answers is not in the document]
Comment (safydy-r) [reply vue.config.js:9]: thanks

**Review by safydy-r — APPROVED** [review]
this lgtm

**Review by object-required — APPROVED** [review]

**Pull request merged**